content/java/concepts/calendar/terms/getAvailableLocales/getAvailableLocales.md

```diff
@@ -0,0 +1,62 @@
+---
+Title: '.getAvailableLocales()'
+Description: 'Returns an array of all locales available in java runtime environment.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Calendar'
+  - 'Methods'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The `.getAvailableLocales()` method of the `Calendar` class is used to return an array of all locales. getAvailableLocals() method is a static method.
+
+## Syntax
+
+```pseudo
+
+public static Locale[] getAvailableLocales()  
+
+```
+
+## Example
+
+The following example displays the available locales.
+
+```java
+
+import java.util.Locale;  
+
+public class JavaCalendargetavailablelocalsExample1 {  
+public static void main(String args[]) {  
+       // an object of local class called strlocal(array) is created   
```

Comment:
```suggestion
       // An object of local class called strlocal(array) is created   
```

Reply:
Comments should begin with a capital and all indentations should use 2 spaces not tabs